diff --git a/android/build.gradle b/android/build.gradle
index e6bcad6d9ab3a9b37f716309c094f41f390c6f2f..b3b322cd60870416764db7ccf926fd9afe3bdc8f 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -54,9 +54,7 @@ def isNewArchitectureEnabled() {
     return project.hasProperty("newArchEnabled") && project.newArchEnabled == "true"
 }
 
-if (isNewArchitectureEnabled()) {
-    apply plugin: 'com.facebook.react'
-}
+apply plugin: 'com.facebook.react'
 
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
